config/clowder_config.rb

```diff
@@ -83,3 +93,12 @@ def self.exists(name, value)
 # ManageIQ Message Client depends on these variables
 ENV["QUEUE_HOST"] = ClowderConfig.queue_host
 ENV["QUEUE_PORT"] = ClowderConfig.queue_port
+
+ENV["DATABASE_HOST"] = ClowderConfig.instance["databaseHostname"]
+ENV["DATABASE_PORT"] = ClowderConfig.instance["databasePort"]
+ENV["CW_AWS_ACCESS_KEY_ID"] = ClowderConfig.instance["awsAccessKeyId"]
+ENV["CW_AWS_SECRET_ACCESS_KEY"] = ClowderConfig.instance["awsSecretAccessKey"]
+ENV["CW_AWS_REGION"] = ClowderConfig.instance["awsRegion"]
+ENV["CLOUD_WATCH_LOG_GROUP"] = ClowderConfig.instance["logGroup"]
+ENV["RAILS_PORT"] = ClowderConfig.instance["webPorts"].to_s
+ENV["RDSCA"] = ClowderConfig.instance["rdsCa"]
```

Comment:
@hsong-rh I think here we should just write out the temp file with the ENV["RDSCA"] data and save the name of the file in an environment variable say called RDSCA_FILE

```
if ClowderConfig.instance["rdsCa"] != nil do
    require 'tempfile'
   file = Tempfile.new('RDSCa.crt')
   ENV["RDSCA_FILE"]=file.path
   file.write(ClowderConfig.instance["rdsCa"])
   file.close
end
```
Then we don't need anything in the entry point script